packages/schema-views/src/schema-view-registry.ts

```diff
@@ -2,18 +2,34 @@ import {Schema} from './schema';
 import {SchemaView} from './schema-view';
 import {SchemaPredicate} from './schema-predicates';
 
+interface IViewRegistration {
+  view: SchemaView;
+  predicate: SchemaPredicate;
+  variant?: string;
+}
+
```

Comment:
We could use a more specific type for the variant, like:

type SchemaViewVariant = 'expanded' | 'summary'

...
variant?: SchemaViewVariant
...